packages/token/src/index.ts

```diff
@@ -19,51 +16,53 @@ type TokenMapping = {
     tokens: TokenMetadata[];
 };
 
-const tokenMappingUrl = process.env.TOKEN_MAPPING_URL || process.env.NEXT_PUBLIC_TOKEN_MAPPING_URL;
+/** The zero address */
+const zeroAddress = "0x0000000000000000000000000000000000000000";
 
-const tokenMappingStr = process.env.NEXT_PUBLIC_TOKEN_MAPPING || process.env.TOKEN_MAPPING;
+/* The base URL for raw token remap files */
+const REMAP_BASE_URL = "https://raw.githubusercontent.com/renegade-fi/token-mappings/main/";
 
-export const tokenMapping: TokenMapping = {
-    tokens: [],
-};
-
-export async function loadTokenMapping() {
-    if (!tokenMappingUrl) {
-        throw new Error("No token mapping URL provided");
-    }
-
-    const res = await fetch(tokenMappingUrl);
-    const data = await res.json();
-    formatTokenMapping(data);
-
-    tokenMapping.tokens = data.tokens;
-}
+/** The list of stablecoins */
+export const STABLECOINS = ["USDC", "USDT"];
 
-function formatTokenMapping(data: any) {
-    for (const t of data.tokens) {
-        t.supported_exchanges = Object.fromEntries(
-            Object.entries(t.supported_exchanges).map(([k, v]) => [
-                // Lowercase all of the exchange names to match the Exchange enum
-                k.toLowerCase(),
-                v,
-            ]),
+/** The token class */
+export class Token {
+    private static tokenMapping: TokenMapping;
+
+    /** Fetch a remap from the repo */
+    static async fetchRemapFromRepo(chain: number) {
+        const name = getSDKConfig(chain).name;
+        const url = new URL(`${REMAP_BASE_URL}${name}.json`);
+        const res = await fetch(url);
+        const data = (await res.json()) as TokenMapping;
+        Token.processRemap(data.tokens);
+    }
+
+    /** Parse a remap from a stringified JSON object */
+    static async parseRemapFromString(remap: string) {
+        const data = JSON.parse(remap) as TokenMapping;
+        Token.processRemap(data.tokens);
+    }
+
+    private static processRemap(tokens: TokenMetadata[]) {
```

Comment:
Don't we also need this to lowercase all exchange names, as `formatTokenMapping` used to?